Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ruby
+
+COPY generate-jenkins-changelog.rb /jenkins-changelog-generator/bin/generate-jenkins-changelog
+COPY lts-backports-changelog.rb /jenkins-changelog-generator/bin/lts-backports-changelog
```

Comment:
This file is being copied but there are no corresponding instructions.

Reply:
"To generate an LTS backports changelog, pass the `-e CHANGELOG_TYPE=lts` variable." in README

Reply:
Got confused because there are two explicit invocations and this is just a side note.

I expected the invocation to be different, but on second thought, it doesn't need to be -- `-e CHANGELOG_TYPE=lts 2.190.3` would suffice.

Reply:
yup. Documentation could be rewritten later